src/main/java/com/basho/riak/client/api/commands/kv/MultiCommand.java

```diff
@@ -54,9 +54,8 @@ protected MultiCommand(Builder builder)
 
         Submitter submitter = new Submitter(operations, maxInFlight, cluster, future);
 
-        Thread t = new Thread(submitter);
-        t.setDaemon(true);
-        t.start();
+        new Thread(submitter).start();
+
```

Comment:
Right ... if there is a `close()` or `stop()` method on the client object, something in the `run()` block needs to know to make an early exit, and, most likely, the threads should be `join()`-ed so that client shutdown can be deterministic. May be more work than it's worth now, so a note could be made and `setDaemon(true)` put back in.